docs/quick_start.md

```diff
@@ -11,7 +13,10 @@
   * [Custom Deployment with Specific ClickHouse Configuration](#custom-deployment-with-specific-clickhouse-configuration)
 
 # Prerequisites
-1. Operational Kubernetes instance
+
+1. Kubernetes instance with the following version considerations:
+    1. `clickhouse-operator` versions **before** `0.16.0` requires [Kubernetes prior to `1.16`](https://kubernetes.io/releases/).
+    1. `clickhouse-operator` versions `0.16.0` **and after** require [Kubernetes version `1.23` and after](https://kubernetes.io/releases/).
```

Comment:
Compatible with kubernetes version 1.22 and after

Reply:
Updated to 1.22.

Reply:
Final words

`clickhouse-operator prior 0.16` compatible with `kubernetes after 1.16 and prior 1.22`
`clickhouse-operator 0.16 and after ` compatible with `kubernetes 1.16 and after`